e2e_playwright/st_slider.py

```diff
@@ -78,3 +85,42 @@ def on_change():
     )
     st.write("Value 8:", st.session_state.slider8)
     st.write("Slider changed:", "slider_changed" in st.session_state)
+
+with st.form(key="my_form", clear_on_submit=True):
+    selection = st.slider(
+        "Label 9",
+        min_value=0,
+        max_value=100,
+        value=25,
+        step=1,
+        key="slider9",
+    )
+    st.form_submit_button("Submit")
+
+st.write("slider-in-form selection:", str(selection))
+
+
+@st.experimental_fragment()
+def test_fragment():
+    selection = st.slider(
+        "Label 10",
+        min_value=0,
+        max_value=100,
+        value=25,
+        step=1,
+        key="slider10",
+        on_change=on_change,
+    )
+    st.write("slider-in-fragment selection:", str(selection))
+
+
+test_fragment()
+
+if "runs" not in st.session_state:
+    st.session_state.runs = 0
+st.session_state.runs += 1
+st.write("Runs:", st.session_state.runs)
+
+st.slider("Slider 11 (formatted float)", value=0.05, step=0.2, format="%d%%")
```

Comment:
Oh my bad, the `%d` cuts of the decimals (see the screenshot). I think using `"%f%%"` should show the decimals correctly